rooms/views.py

```diff
@@ -1,3 +1,60 @@
-from django.shortcuts import render
+import json, datetime
 
-# Create your views here.
+from django.http      import JsonResponse
+from django.views     import View
+from django.db.models import Avg, Count
+
+from .models import *
```

Comment:
```suggestion
from .models import Room
```
실제 코드에서는 *을 사용하지 말고 사용하는 것들을 직접 입력하세요.

Reply:
네엡! 😊